CHANGELOG.md

```diff
@@ -90,6 +90,8 @@ and this project adheres to
   - [#4194](https://github.com/bpftrace/bpftrace/pull/4291)
 - Only cache symbols from targeted process
   - [#4315](https://github.com/bpftrace/bpftrace/pull/4315)
+- Refactor naming of Probe objects
```

Comment:
Nit: I think this qualifies as a fix but NBD.

Reply:
Fixed